### ISSUE TITLE
Create CNN to predict rectangle for an image

### ISSUE THREAD

**Issue by himanshub16** [sem6-project/grasp-detector#3]
Create a CNN which takes an image as input and predicts a vector of dimension 5. The vector represents the rectangle `(X, Y, width, height, tilt_angle)`.

I tried to create a CNN in Keras but failed.

**Comment by faheemzunjani** [comment]
My main concern here is that normal CNNs are only used for object classification so they usually only output class labels for the given image. I think CNN might not be the right choice to predict 'a rectangle.'

**Comment by himanshub16** [comment]
https://arxiv.org/abs/1412.3128

https://arxiv.org/abs/1611.08036

**Comment by adityapowale** [comment]
x,y are taken to be centre of rectangle in the paper.

**Comment by invokesus** [comment]
Closing since Pytorch implementation is complete.
